Makefile

```diff
@@ -5,16 +5,21 @@ VERSION_IMPORT_PATH = github.com/elastic/elastic-package/internal/version
 .PHONY: build
 
 build:
-	go install -ldflags \
+	go build -ldflags \
 	    "-X $(VERSION_IMPORT_PATH).CommitHash=`git describe --always --long --dirty` -X $(VERSION_IMPORT_PATH).BuildTime=`date +%s` -X $(VERSION_IMPORT_PATH).Tag=`(git describe --exact-match --tags 2>/dev/null || echo '') | tr -d '\n'`" \
```

Comment:
Can this string be factored out since it appears twice?

Reply:
In general, there is a [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)), but if it isn't a big deal, we can extract this line to the top.

Reply:
I extracted and split the line to give it more clarity in [8182d2f](https://github.com/elastic/elastic-package/pull/663/commits/8182d2f5e5b70b1a1c9b7611187a72a615fdd588)

Reply:
What you have now is very nice. Thanks.